Specifying that Python is exclusively one-liners (#1)
exterkamp/exterkamp: README.md

* [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/)
* [Chrome UX Report](https://developers.google.com/web/tools/chrome-user-experience-report)

> These days I write mostly Go, Javascript, and C++; with some Python one-liners & Java in the past.

Reviewer: Surely this shouldn't be "some" if it's for one-liners?

```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & Java in the past.
```

> These days I write mostly Go, Javascript, and C++; with many Python one-liners & some Java in the past.

Reviewer: ```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past.
```

> These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past. I spend my free time playing Kane & Lynch 2.

Reviewer: ```suggestion
These days I write mostly Go, Javascript, and C++; with many Python one-liners & a bit of Java in the past.
```
